apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -14,48 +16,62 @@ import {
   ONE_TIME_DIVIDEND,
   RECURRING_DIVIDEND,
   ONE_TIME_MERIT,
+  READY,
+  CLAIMED,
 } from '../../utils/constants'
 import { Empty } from '../Card'
 import Metrics from './Metrics'
-import { useAppState } from '@aragon/api-react'
+import { useAppState, useAragonApi, useNetwork } from '@aragon/api-react'
```

Comment:
Can I remove the `useAppState` hook and import `appState` from `useAragonApi`? I've seen it elsewhere but I'm not 100% sure it's the same thing.

Reply:
[Yes](https://hack.aragon.org/docs/api-js-ref-react#useappstate) 😄

They are the same, and I think I prefer always using `useAragonApi`